osu.Game/Modes/UI/HudOverlay.cs

```diff
@@ -94,6 +115,23 @@ private void load(OsuConfigManager config, NotificationManager notificationManag
             }
         }
 
+        public void ForceProgressVisibility()

```

Comment:
in fact what the hell? why is this in `HudOverlay`? you're doing code it in the wrong class again.